package/python-kiwi-spec-template

```diff
@@ -601,17 +610,23 @@ Group:          %{sysgroup}
 Provides manual pages to describe the kiwi commands
 
 %prep
-%setup -q -n kiwi-%{version}
+%autosetup -n kiwi-%{version}
 
 # Drop shebang for kiwi/xml_parse.py, as we don't intend to use it
 # as an independent script
 sed -e "s|#!/usr/bin/env python||" -i kiwi/xml_parse.py
 
+# Build documentation
+make -C doc man
+
+# Build application wheel
+python3 -m build --no-isolation --wheel
```

Comment:
Doesn't this work?
```suggestion
# Build application wheel
%pyproject_wheel
```

Reply:
We can't use that because the macro doesn't exist for Debian/Ubuntu yet. I haven't ported the pyproject macros to Debian distributions just yet.